rtv/terminal.py

```diff
@@ -26,6 +32,9 @@
     from six.moves import html_parser
     unescape = html_parser.HTMLParser().unescape
 
+# programs that don't support open file by url
+black_list_programs = ['gpicview.desktop', 'pinta.desktop']
```

Comment:
Where did this list come from. I'm worried that if these are just the ones that you've found so far, there could be dozens of other programs that need to be blacklisted.